src/psyclone/tests/psyir/backend/c_test.py

```diff
@@ -324,8 +324,8 @@ def test_cw_binaryoperator():
     binary_operation = BinaryOperation(BinaryOperation.Operator.ADD)
```

Comment:
Please could you fix the two complains about class `Unsupported` inheriting from object.

Reply:
Done